test/unit/state/siteUtilTest.js

```diff
@@ -286,6 +286,14 @@ describe('siteUtil', function () {
           assert.equal(!!processedSites.getIn([0, 'lastAccessedTime']), true)
           assert.deepEqual(processedSites.getIn([0, 'tags']).toJS(), [])
         })
+        it('returns true for bookmarked entries', function () {
+          const siteDetail = Immutable.fromJS({
+            location: testUrl1,
+            lastAccessedTime: 123,
+            tags: [siteTags.BOOKMARK]
+          })
```

Comment:
@cezaraugusto the above mention by @darkdh is correct; we need to actually call `addSite` (which then calls `mergeSiteDetails`, where the change was made).  When you call `addSite`, you'll want to make a test case for each value.  Where it returns:
- lastAccessedTime = newSiteDetail.get('lastAccessedTime')
- lastAccessedTime = oldSiteDetail.get('lastAccessedTime')
- lastAccessedTime = 0